examples/golang-todo-rest-crud/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build stage
+FROM golang:1.19.4-alpine3.17 AS builder
+WORKDIR /build
+
+# Copy module files first so that they don't need to be downloaded again if no change
+COPY go.* ./
+RUN go mod download
+RUN go mod verify
+
+# Copy source files and build the binary
+COPY . .
+RUN go build -o todo ./cmd/todo
+
+# Run stage
+FROM alpine:3.17
+WORKDIR /build
+COPY --from=builder /build/todo .
+COPY --from=builder /build/.env .
+COPY --from=builder /build/third_party/swagger-ui-4.11.1 /build/third_party/swagger-ui-4.11.1
+
+CMD ["/build/todo"]
```

Comment:
We need to make this Dockerfile to run as a nonroot user. For that you can follow the following guide to make it work:

[Official Docker documentation on how to build Go images](https://docs.docker.com/language/golang/build-images/#multi-stage-builds) (recommended approach). Uses a distroless image that already takes care of this!

Reply:
✔️ Done! I have to updated the golang image to buster to make it work